setcontext/setcontext.py

```diff
@@ -0,0 +1,406 @@
+
+import fire
+import functools
+import json
+import os
+import re
+import pathlib
+import subprocess
+import sys
+from termcolor import cprint
+
+
+#Add pytest
+#Build framework for  commandline tools.
+
+"""
+Global variable for local git repo.
```

Comment:
Since it is not a multi line comment, use `# Global ...`

Reply:
Done